<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>projCreationWindow</class>
 <widget class="QWidget" name="projCreationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a Project</string>
  </property>
  <widget class="QLabel" name="lbl_Keywords">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Keywords</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Title">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_AddLanguage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_FilmLocs">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>370</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_Languages">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_ReleaseDate">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dd/M/yy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2000</year>
     <month>10</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QListWidget" name="lst_Keywords">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>410</y>
     <width>181</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_Title">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>113</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_Genre">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>113</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_Summary">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>601</width>
     <height>161</height>
    </rect>
   </property>
   <property name="documentTitle">
    <string/>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>A Description of the Movie</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lst_FilmLocs">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>410</y>
     <width>181</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_FilmLocs">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Filming locations</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Runtime">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Runtime</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_DelKeyword">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>610</y>
     <width>51</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_AddFilmLoc">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>610</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_TicketSale">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>67</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Weekly Ticket Sale</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_RlsDate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Release Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Genre">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Genre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_TicketSale">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>113</width>
     <height>29</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Eg: $1.00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_AddKeyword">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>610</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_DelFilmLoc">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>610</y>
     <width>51</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_DelLanguauges">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>610</y>
     <width>51</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_SaveDetails">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>660</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_Keyword">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>370</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Summary">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Summary</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_RunTime">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>113</width>
     <height>29</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>SS-MM-HH</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lst_Languages">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>181</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Languages">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Languages</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_AddMaterial">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Material</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Img">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>26</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_SelectImg">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>120</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Poster</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
